python/MANIFEST

```diff
@@ -9,13 +9,15 @@
 ./pdo/contract/code.py
 ./pdo/contract/contract.py
 ./pdo/contract/message.py
+./pdo/contract/replication.py
 ./pdo/contract/request.py
```

Comment:
Can I request you to add ./pdo/contract/transaction.py also to the MANIFEST?

Reply:
not my file. :-)
done.